packages/manager/apps/web/client/app/hosting/multisite/git-integration/removal/git-removal.service.js

```diff
@@ -0,0 +1,16 @@
+export default class HostingMultisiteGitRemovalService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  deleteGitAssociation(serviceName, path, deleteFiles = false) {
+    return this.$http
+      .get(`/hosting/web/${serviceName}/website?path=${path}`)
+      .then(({ data: [domainId] }) => {
+        this.$http.delete(
+          `/hosting/web/${serviceName}/website/${domainId}?deleteFiles=${deleteFiles}`,
+        );
+      });
+  }
```

Comment:
I think this call should be splited, because we already have a function for the get so on it should be only the delete call

Reply:
It will make dependency beteween the deleteGitModule  and the associationGitModule ?

**Now:**
deleteModule => deleteService

**After:**
deleteModule => deleteService, associationService (from associationGitModule)

 Not sure it is better no?